config/config_encryption.go

```diff
@@ -41,6 +42,11 @@ func promptForConfigEncryption() (bool, error) {
 	return common.YesOrNo(input), nil
 }
 
+// Unencrypted provides the default key provider implementation for unencrypted files
+func Unencrypted() ([]byte, error) {
```

Comment:
The only place this is used is in tests and seems like it could just be made inline rather than added the GCT binary on compilation

Reply:
Yep, I thought about that as well, but decided to include it in the config package (rather than just test file) for convenience for the "other consumers" of the `ReadConfig` method. If you disagree, I'm OK to remove, but I do prefer to keep it. See next comment.